stripe/functions/index.js

```diff
@@ -63,6 +63,22 @@ exports.createStripeCustomer = functions.auth.user().onCreate(event => {
   });
 });
 
+// Add a payment source (card) for a user by writing a stripe payment source token to Realtime database
+exports.addPaymentSource = functions.database.ref('/stripe_customers/{userId}/cards/{token}').onWrite(event => {
+  const token = event.data.val();
+  const user = event.params.userId;
```

Comment:
[Isn't this the location](https://github.com/firebase/functions-samples/blob/e9490d7e8df49a53eb9da44e5865044dc7f90aba/stripe/functions/index.js#L68) in your schema where you are currently storing a Stripe customer id?